### PR TITLE
Lesson 2 Level 3 done

### PR DESCRIPTION
**Pull request by kskaretnikov**

### CODE REVIEW

File: src/main/java/students/student_konstantin_karetnikov/AveArithm.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+        System.out.println("Answer is = " + ((firstNumber + secondNumber + thirdNumber) / 3));
+
+
+
+
+    }
+
+    }
```
Comment (antdolg1) [inline src/main/java/students/student_konstantin_karetnikov/AveArithm.java:27]: Format code and remove redundant empty lines.

File: src/main/java/students/student_konstantin_karetnikov/MltplTable.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        System.out.println(number + " x 5 = " + (number * 5));
+        System.out.println(number + " x 6 = " + (number * 6));
+        System.out.println(number + " x 7 = " + (number * 7));
+        System.out.println(number + " x 8 = " + (number * 8));
+        System.out.println(number + " x 9 = " + (number * 9));
+        System.out.println(number + " x 10 = " + (number * 10));
+
+    }
+
+    }
```
Comment (antdolg1) [inline src/main/java/students/student_konstantin_karetnikov/MltplTable.java:25]: Same here, format and remove empty lines.

File: src/main/java/students/student_konstantin_karetnikov/PerimeterArea.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package students.student_konstantin_karetnikov;
+import java.util.Scanner;
+
+public class PerimeterArea {
+    public static void main(String[] args) {
+        double Number;
```
Comment (antdolg1) [inline src/main/java/students/student_konstantin_karetnikov/PerimeterArea.java:6]: Variable name cannot start from capital letter.

Also format code and remove empty lines.

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Changes needed.

**Comment by kskaretnikov** [comment]
Changed

**Review by antdolg1 — APPROVED** [review]

**Pull request merged**